drivers/manual_hmis_data/app/views/manual_hmis_data/inventories/edit.haml

```diff
@@ -1,5 +1,5 @@
 = content_for :modal_title do
-  Edit Inventory for #{@inventory.project.name} at #{@inventory.project.organization.name}
+  Edit Inventory for #{@inventory.project.name(current_user)} at #{@inventory.project.organization.name}
```

Comment:
I _think_ you need `can_edit_projects?` to get to this page.  Perhaps we should pass that in as well?

Reply:
updated to ignore confidential status